SU-SWS/stanford_basic
# D8CORE-2562: Update option A line-height to prevent overlap.

# READY FOR REVIEW 

# Summary
- Updates line-height on option-A line 1 so it can wrap to multiple lines.

# Review By (Date)
- Sept 2nd

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Clear all caches
3. Enter "long way to Tipperary. Long long way to go." as line one in the lockup settings of Option - A
4. Save and view on multiple screen sizes.

# Affected Projects or Products
- D8CORE
- Decanter

# Associated Issues and/or People
- D8CORE-2562
- https://github.com/SU-SWS/decanter/pull/722

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
